specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-02-01/examples/AgentPoolsCreate_WindowsDisableOutboundNAT.json

```diff
@@ -0,0 +1,63 @@
+{
+  "parameters": {
+    "api-version": "2024-02-01",
+    "subscriptionId": "subid1",
```

Comment:
Please fix failed CI [Swagger ModelValidation](https://github.com/Azure/azure-rest-api-specs/pull/27872/checks?check_run_id=21851951618).

> Object didn't pass validation for format uuid: subid1
> Url: [aks/stable/2024-02-01/managedClusters.json#L632:32](https://github.com/Azure/azure-rest-api-specs/blob/f8cf7d9c1c19550bf1c77bdc17d7efe8b71e4733/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-02-01/managedClusters.json#L632:32)
> ExampleUrl: [stable/2024-02-01/examples/AgentPoolsCreate_WindowsDisableOutboundNAT.json#L4:23](https://github.com/Azure/azure-rest-api-specs/blob/f8cf7d9c1c19550bf1c77bdc17d7efe8b71e4733/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-02-01/examples/AgentPoolsCreate_WindowsDisableOutboundNAT.json#L4:23)

Reply:
Thank you @FumingZhang  . Updated.